src/main/java/net/dv8tion/jda/api/entities/Guild.java

```diff
@@ -2356,6 +2356,17 @@ default Emote getEmoteById(long id)
     @Nonnull
     ExplicitContentLevel getExplicitContentLevel();
 
+    /**
+     * Returns the NSFW Level that this guild is classified with.
+     * <br>For a short description of the different values, see {@link net.dv8tion.jda.api.entities.Guild.NSFWLevel NSFW Level}.
+     * <p>
+     * This value cannot be modified directly.
```

Comment:
When we have more information we should indicate _how_ this value gets changed if developers can't change it themselves given the fact that channel specific `nsfw` is modifyable.